openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1729,7 +1729,7 @@ The following additional Credential Issuer metadata are defined for this Credent
 * `credential_definition`: REQUIRED. Object containing the detailed description of the Credential type. It consists at least of the following two sub claims:
   * `type`: REQUIRED. Array designating the types a certain Credential type supports according to [@VC_DATA], Section 4.3.
   * `credentialSubject`: OPTIONAL. An object containing a list of name/value pairs, where each name identifies a claim offered in the Credential. The value can be another such object (nested data structures), or an array of such objects. To express the specifics about the claim, the most deeply nested value MAY be an object that includes a following non-exhaustive list of parameters defined by this specification:
-      * `mandatory`: OPTIONAL. Boolean which when set to `true` indicates the claim MUST be present in the issued Credential. If the `mandatory` property is omitted its default should be assumed to be `false`.
+      * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the Credential Issuer will always include this claim in the issued Credential. If the `mandatory` property is omitted its default should be assumed to be `false`.
```

Comment:
```suggestion
      * `mandatory`: OPTIONAL. Boolean which, when set to `true`, it signifies that the claim is always included in the issued Credential. If set to `false`, the Issuer can choose not to include the claim in certain cases. If the `mandatory` parameter is omitted, the default value is `false`.
```

Reply:
```suggestion
      * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the Credential Issuer will always include this claim in the issued Credential. If set to `false`, the Credential Issuer can choose not to include the claim. If the `mandatory` parameter is omitted, the default should be assumed to be `false`.
```

Reply:
"the Credential Issuer can choose not to include the claim" potentially seems to suggesting the decision is only made by the issuer; I was presuming the decision could also be made by the user or based on what the wallet requests?

Perhaps something more like "the Credential Issuer may issue Credentials that do not include the claim" ?

Reply:
That is why I thought we would need to be more precise in the first place. If the issuer does not have the info, the issuer cannot return it and the wallet should not fail.

Reply:
```suggestion
 * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the Credential Issuer will always include this claim in the issued Credential. If set to `false`, the claim is not included in the issued Credential if the wallet did not request the inclusion of the claim in the Authorization Request, or if the Issuer chose to not include the claim. If the `mandatory` parameter is omitted, the default should be assumed to be `false`.
```